resilience4j-retrofit/README.adoc

```diff
@@ -0,0 +1,39 @@
+= Retrofit Circuitbreaker
+
+https://square.github.io/retrofit/[Retrofit] client circuit breaking
+using
+https://github.com/robwin/javaslang-circuitbreaker[Javaslang-circuitbreaker]
+
```

Comment:
Created a `retrofit.adoc` file for the module in the resilience4j-documentation project.  I've not linked into the index for the moment, I guess you would want some kind of 'extension modules' section to the document.